client/src/app/page/add-task-form/add-task-form.component.scss

```diff
@@ -35,13 +55,37 @@
 .form-wrapper a:hover,
 .form-wrapper a:active {
   color: $primary-color;
+  text-decoration: none;
 }
 
 .form-wrapper a {
-  text-decoration: underline;
+  position: relative;
+  display: inline-block;
+
+  &:before {
+    position: absolute;
+    bottom: 0;
+    display: block;
+    width: 100%;
+    height: 1px;
+    background: $primary-color;
+    content: "";
+  }
 }
 
 .form-wrapper a:hover,
 .form-wrapper a:active {
-  text-decoration: none;
+
+  &:before {
+    background: transparent;
+  }
+}
+
+.pos-absolute {
+  position: absolute;
+  z-index: 100;
+}
+
+.hide-true {
```

Comment:
it's not good.
use class .hidden for example
and use it like this:
ng-class="{'class-one' : value.one , 'class-two' : value.two}"